library/nats/2.10/Kraftfile

```diff
@@ -0,0 +1,151 @@
+spec: v0.6
+
+name: nats
+
+rootfs: ./Dockerfile
+
+cmd: [
+  "/usr/bin/nats-server",
+  "-a",
+  "172.44.0.2",
```

Comment:
Can you use maybe `0.0.0.0`, this IP will not be fixed

Reply:
Using `0.0.0.0` will cause the error message "[1] 2025/03/19 13:58:38.081344 [ERR] Address "0.0.0.0" can not be resolved properly", but the server will still be running correctly. I think the reason is that `0.0.0.0` is used to bind to all interfaces, but Unikraft does not handle it correctly. If it's find with the message showing up, I can just remove the argument.